src/components/Donation/machines/donationMachine.ts

```diff
@@ -22,12 +22,12 @@ const donationWidgetMachine = Machine<
     context: {
       donation: {
         message: '',
-        status: ''
+        status: '',
+        errors: null
```

Comment:
In the membership machine I have this `errors` key in singular. Can we unify both of them?
_Not sure why you choose the plural but if you are certain about it then let's keep both plural..._

Having the same distribution on both machines will make a better DX